spec/results-model-spec.js

```diff
@@ -33,6 +35,10 @@ describe("ResultsModel", () => {
       expect(resultsModel.getPathCount()).toBe(1);
       expect(resultsModel.getMatchCount()).toBe(6);
       expect(resultsModel.getPaths()).toEqual([editor.getPath()]);
+      if (parseFloat(atom.getVersion()) >= 1.17) {
```

Comment:
I still think it wouldn't matter given that find-and-replace is a core package and is tied to specific Atom versions.